Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -20,7 +20,7 @@ constexpr u16 TCP_FLAG_PSH = 0x8;
 constexpr u16 TCP_FLAG_FIN = 0x1;
 constexpr u16 TCP_FLAG_RST = 0x4;
 
-constexpr u16 IP_PROTOCOL = 0x800;
+constexpr u16 DOLPHIN_IP_PROTOCOL = 0x800;
 constexpr u16 ARP_PROTOCOL = 0x806;
```

Comment:
Can't these constants be removed and `Common::IPV4_ETHERTYPE` and `Common::ARP_ETHERTYPE` from `Common/Network.h` be used instead?

Reply:
That sounds like a better idea to me, yeah.

Reply:
Ok feel free to fix separately. This PR was more to bring light to the issue and provide a workaround to users looking for a fix.